web/src/tippyjs.js

```diff
@@ -130,7 +130,10 @@ export function initialize() {
                 const local_id = $elem.attr("data-reaction-id");
                 const message_id = rows.get_message_id(instance.reference);
                 const title = reactions.get_reaction_title_data(message_id, local_id);
-                instance.setContent(title);
+                instance.setContent(parse_html(title));
+                instance.popper
+                    .querySelector(".tippy-content > div > span")
+                    .classList.add("break_all_words");
```

Comment:
Rather than adding the CSS class here, we should be adding it when we generate the actual spans, I think; it's way easier to understand that way.